bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/VBF_HH/VBF_HH_CV_0_5_C2V_1_C3_1_14TeV-madgraph/VBF_HH_CV_0_5_C2V_1_C3_1_14TeV-madgraph_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model SM_HEL_UFO_noLightYukawa
+define p = u c d s u~ c~ d~ s~
+define j = p
+import model SM_HEL_UFO_noLightYukawa
+generate p p > h h j j $$ z w+ w-  / a j QED=4
```

Comment:
Why do you need "/ a j" here?
With and without "/ a j" both gives me same number of processes and diagrams

92 processes with 432 diagrams generated in 2.689 s
Total: 92 processes with 432 diagrams